### DISCUSSION TITLE
安装DeepSpeech后虚拟环境没有PaddlePaddle

### DISCUSSION THREAD

**Discussion by liweigu** [PaddlePaddle/PaddleSpeech#765] [Q&A] [answered]
先安装了PaddlePaddle，再安装DeepSpeech，后者会创建virtualenv虚拟环境，然后在运行“python3 -u ./deepspeech/exps/deepspeech2/bin/deploy/server.py ...”时提示： No module named 'paddle'
看起来是在虚拟环境中没有前面安装的PaddlePaddle了，请问应该怎么设置环境后才能使用，还是安装时要修改什么步骤呢？

**Comment by liweigu** [comment]
修改脚本全程不使用 virtualenv 后，运行时提示错误为 No module named 'deepspeech'

**Comment by liweigu** [comment, marked as answer]
可以运行 server.py 了，但是必须执行：
cp -r deepspeech/models/ds2/ deepspeech/models/deepspeech2
提交了issue： https://github.com/PaddlePaddle/DeepSpeech/issues/770

**Reply by zh794390558** [reply]
多谢，这部分修改没有改全。
